src/components/structures/RightPanel.tsx

```
@@ -302,8 +310,9 @@ export default class RightPanel extends React.Component<Props, IState> {
                 break;
 
             case RightPanelPhases.Widget:
+             
                 if (!!this.props.room && !!cardState?.widgetId) {
-                    card = <WidgetCard room={this.props.room} widgetId={cardState.widgetId} onClose={this.onClose} />;
+                    card = <WidgetCard room={this.props.room} widgetId={cardState.widgetId} onClose={this.onClose}  windowWidth={this.state.windowWidth}/>;
```

ESLint

[error] A space is required before closing bracket

(react/jsx-tag-spacing)